projects/gui/src/mainwindow.cpp

```diff
@@ -565,7 +565,15 @@ void MainWindow::setCurrentGame(const TabData& gameData)
 	}
 
 	// if a human plays the black side against an engine then flip the board
-	if (m_players[Chess::Side::Black]->isHuman() && !m_players[Chess::Side::White]->isHuman())
+	if (m_players[Chess::Side::Black]->isHuman()
+	&& !m_players[Chess::Side::White]->isHuman())
+		m_gameViewer->boardScene()->flip();
+
+	// flip the board if gauntlet player 0 has black pieces
+	else if (gameData.tournament
+	&&  gameData.tournament->type() == "gauntlet"
+	&&  gameData.tournament->playerAt(0).name() == m_players[Chess::Side::Black]->name()
```

Comment:
I'd prefer to compare player indexes instead of names if possible, eg:

```
// Check if the black player is the gauntlet player
if (gameData.tournament->m_gameData[m_game]->blackIndex == 0)
```

Of course `Tournament::m_gameData` is private so there needs to be a new helper method, eg. `int Tournament::playerIndex(ChessGame* game, Chess::Side side) const`.